services/movie.py

```diff
@@ -0,0 +1,42 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if not genres_ids and not actors_ids:
+        return queryset
+
+    if genres_ids and actors_ids:
+        return queryset.filter(genres__in=genres_ids, actors__in=actors_ids)
+
+    if genres_ids:
+        return queryset.filter(genres__in=genres_ids)
+
+    if actors_ids:
+        return queryset.filter(actors__in=actors_ids)
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie | str:
```

Comment:
The return type of the function `get_movie_by_id` is inconsistent. It's better to return `None` or raise an exception when the movie does not exist, instead of returning a string. This way, the function will always return a `Movie` instance or `None`, which makes it easier to handle the return value.